geode-core/src/main/java/com/gemstone/gemfire/internal/cache/LocalRegion.java

```diff
@@ -17,10 +17,96 @@
 
 package com.gemstone.gemfire.internal.cache;
 
-import com.gemstone.gemfire.*;
```

Comment:
If you want to clean up these imports you should not do it as part of this fix. File a new ticket for it. I could be wrong about this but I think you should change your IDE to not automatically reorg the imports. Kirk might have some good advice on this.